### ISSUE TITLE
A11y_.NET Core_MAUI_Recipe Search_Keyboard : The Recipe Items are not able to access using keyboard. 

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [dotnet/maui#15599]
### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[MAUI]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG2.1.1;#DesktopApp;#Win11;#Benchmark;

### Actual:
The recipe Items when searched and tried to access via keyboard Enter or Space bar key , it is not getting activated.

### Expected:
The Recipe items must be accessed by using keyboard Enter or Space Key.

### User Impact:
Keyboard Users will face issue while accessing the controls/buttons.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. Tab till "My Recipes" and select.
3. Tab till any existing recipe and select.
4. "Recipe" Screen will open.
5. Tab till any recipe and observe the issue by hitting enter on it.

### Attachment:
[Recording-20230608_103001.webm](https://github.com/dotnet/maui/assets/85863895/7cc35ee4-6302-43f6-b4ba-40bfc45a3ce6)

### Link to public reproduction project repository

Test

### Version with bug

7.0.49

### Last version that worked well

7.0

### Affected platforms

Windows

### Affected platform versions

Windows 11 23H2

**Comment by PoojaNamde** [comment]
#A11ypreview;

Added A11ypreview tag

**Comment by PoojaNamde** [comment]
Verified this issue in latest environment, but issue is still able to reproduce.

**Comment by merriemcgaw** [comment]
@PoojaNamde did you update to the latest MAUI bits? @rachelkang you may need to provide an updated SDK.

**Comment by PoojaNamde** [comment]
@merriemcgaw Updated to latest MAUI App but still able to repro this issue.

**Comment by rachelkang** [comment]
Hi, @PoojaNamde - could you please confirm that you are testing on:
- Visual Studio Community 2022 Preview, version 17.8.0 Preview 2.0 or later?
- https://github.com/rachelkang/recipesearch **main** branch? (I realize the initial instructions said to test on a different branch, so apologies for any confusion here).

To access the recipe items via keyboard, you can:
- Tab to the collection
- use Arrow Keys to navigation between the items
- press Enter to achieve the same effect as "click"ing on an item and navigating to the next page

Here is a video showing the fix below:
https://github.com/dotnet/maui/assets/21988533/aaed209c-d67e-4733-86d2-45247fed5cc6

If you have different expectations and/or are still not seeing this fix, please let me know. Thank you!

**Comment by PoojaNamde** [comment]
@rachelkang 

We are using Microsoft Visual Studio Community 2022 (64-bit) - Preview Version 17.8.0 Preview 2.0. We have downloaded the given build but while running it is showing below errors. Could you please look into this and help us to resolve it.

![image](https://github.com/dotnet/maui/assets/90901519/39f8ab74-b82f-4512-b904-6cd0bd4caeba)

**Comment by rachelkang** [comment]
Hi, @PoojaNamde - could you please share with me the errors that show up in the Output tab?

**Comment by PoojaNamde** [comment]
@rachelkang   please find the below error that we are getting.
Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1102	Unable to find package Microsoft.NETCore.App.Runtime.Mono.android-arm with version (= 8.0.0-rc.2.23476.7)
  - Found 58 version(s) in nuget.org [ Nearest version: 8.0.0-rc.1.23419.4 ]
  - Found 0 version(s) in Microsoft Visual Studio Offline Packages
  - Found 0 version(s) in C:\Program Files\dotnet\library-packs	Recipes	C:\Users\v-poojanamde\Downloads\recipeSearch-main\recipeSearch-main\WhatToEat\Recipes.csproj	1	

Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1102	Unable to find package Microsoft.NETCore.App.Runtime.Mono.android-x86 with version (= 8.0.0-rc.2.23476.7)
  - Found 58 version(s) in nuget.org [ Nearest version: 8.0.0-rc.1.23419.4 ]
  - Found 0 version(s) in Microsoft Visual Studio Offline Packages
  - Found 0 version(s) in C:\Program Files\dotnet\library-packs	Recipes	C:\Users\v-poojanamde\Downloads\recipeSearch-main\recipeSearch-main\WhatToEat\Recipes.csproj	1	

Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1102	Unable to find package Microsoft.NETCore.App.Runtime.Mono.android-x64 with version (= 8.0.0-rc.2.23476.7)
  - Found 58 version(s) in nuget.org [ Nearest version: 8.0.0-rc.1.23419.4 ]
  - Found 0 version(s) in Microsoft Visual Studio Offline Packages
  - Found 0 version(s) in C:\Program Files\dotnet\library-packs	Recipes	C:\Users\v-poojanamde\Downloads\recipeSearch-main\recipeSearch-main\WhatToEat\Recipes.csproj	1	

Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1102	Unable to find package Microsoft.NETCore.App.Runtime.Mono.android-arm64 with version (= 8.0.0-rc.2.23476.7)
  - Found 58 version(s) in nuget.org [ Nearest version: 8.0.0-rc.1.23419.4 ]
  - Found 0 version(s) in Microsoft Visual Studio Offline Packages
  - Found 0 version(s) in C:\Program Files\dotnet\library-packs	Recipes	C:\Users\v-poojanamde\Downloads\recipeSearch-main\recipeSearch-main\WhatToEat\Recipes.csproj	1	

Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1102	Unable to find package Microsoft.NETCore.App.Runtime.Mono.iossimulator-x64 with version (= 8.0.0-rc.2.23476.7)
  - Found 58 version(s) in nuget.org [ Nearest version: 8.0.0-rc.1.23419.4 ]
  - Found 0 version(s) in Microsoft Visual Studio Offline Packages
  - Found 0 version(s) in C:\Program Files\dotnet\library-packs	Recipes	C:\Users\v-poojanamde\Downloads\recipeSearch-main\recipeSearch-main\WhatToEat\Recipes.csproj	1	

Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1102	Unable to find package Microsoft.NETCore.App.Runtime.Mono.maccatalyst-x64 with version (= 8.0.0-rc.2.23476.7)
  - Found 58 version(s) in nuget.org [ Nearest version: 8.0.0-rc.1.23419.4 ]
  - Found 0 version(s) in Microsoft Visual Studio Offline Packages
  - Found 0 version(s) in C:\Program Files\dotnet\library-packs	Recipes	C:\Users\v-poojanamde\Downloads\recipeSearch-main\recipeSearch-main\WhatToEat\Recipes.csproj	1

**Comment by rachelkang** [comment]
Hi, @PoojaNamde - I've pushed up a new branch that pertains to Windows only, so hopefully you won't run into these irrelevant issues. Could you pull down the latest from https://github.com/rachelkang/recipeSearch/tree/windows-net8 and test on the `windows-net8` branch? 

If you continue to have issues, please let me know.

**Comment by PoojaNamde** [comment]
@rachelkang Could you please close this bug as I am not able to close it.

#Regressed:10-16-23; #Closed;

Verified this issue in the provided env. Issue is fixed hence closing the bug.


https://github.com/dotnet/maui/assets/90901519/aae33f3f-63b8-47e2-bd0a-52a20117bcaf

**Comment by merriemcgaw** [comment]
[heart]         Merrie McGaw reacted to your message:
________________________________
